acq4/modules/TaskRunner/analysisModules/Imaging/Imaging.py

```diff
@@ -37,13 +40,26 @@ def __init__(self, *args):
             dict(name='downsample', type='int', value=1, suffix='x', bounds=[1,None]),
             dict(name='display', type='bool', value=True),
             dict(name='scanProgram', type='list', values=[]),
+            dict(name='Objective', type='str', value='Unknown', readonly=True),
+            dict(name='Filter', type='str', value='Unknown', readonly=True),
             ])
         self.ptree.setParameters(self.params, showTop=False)
         self.params.sigTreeStateChanged.connect(self.update)
         self.params.child('detectors').sigAddNew.connect(self.addDetectorClicked)
         self.params.child('decomb', 'auto').sigActivated.connect(self.autoDecomb)
 
         self.man = getManager()
+        print self.man, dir(self.man.devices)
+        self.scannerDev = self.man.getDevice('Scanner')
+        # find first scope device that is parent of scanner
+        dev = self.scannerDev
+        while dev is not None and not isinstance(dev, Microscope):
+            dev = dev.parentDevice()
+        self.scopeDev = dev
+        self.filterwheel = self.man.getDevice('FilterWheel')
```

Comment:
I added a PMT device class and this class returns a parent FilterDevice. I used the same approach to implement the FilterDevice in the Imager module. 

What do you think of the implementation?